openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2293,6 +2268,178 @@ The following is a non-normative example of a Credential Response containing a C
 
 <{{examples/credential_response_sd_jwt_vc.txt}}
 
+# Claims Description 
+
+Claims description objects are used in two places in this specification, in the
+Credential Issuer metadata and in the Authorization Details. The following
+sections define the structure and semantics of claims description objects.
+
+## Claims Description for Authorization Details {#claims-description-authorization-details}
+
+A claims description object as used in authorization details is an object that defines the
+requirements for the claims that the Wallet requests to be included in the
+Credential.
+
+For a claims description object, the following keys defined by this
+specification can be used to describe the claim or claims (other keys MAY also
+be used):
+
+  * `path`: REQUIRED. Array. Claim path pointer as defined in (#claims_path_pointer)
+    to identify the claim(s) in the Credential.
+  * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the
+    Wallet will only accept a Credential that includes this claim. If set to
+    `false`, the claim is not required to be included in the Credential. If the
+    `mandatory` parameter is omitted, the default value is `false`.
+
+The rules defined in (##claims-description-processing) apply.
+
+## Claims Description for Issuer Metadata {#claims-description-issuer-metadata}
+
+A claims description object as used in the Credential Issuer metadata is an
+object used to describe how a certain claim in the Credential should be
+displayed to the End-User. It is used in the `claims`
+parameter in the Credential Issuer metadata defined in (#format-profiles). The
+following keys can be used to describe the claim or claims:
+
+  * `path`: REQUIRED if the Credential Format uses a JSON-based claims
+    structure; MUST NOT be present otherwise. The value MUST be a non-empty
```

Comment:
I think this is discriminatory to Credential Formats using YAML based claims structures... ;-)

More seriously I do wonder if this goes a little too far with the normative language - it seems unnecessary to completely prevent someone defining a new non-JSON hierarchical credential format from using the `path` key for example.

Reply:
will be resolved with the below suggestion